sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -383,19 +383,97 @@
+    "\n",
+    "Automatic sweeps are triggered by setting `max_trials` to a value greater than 1 in limits and by omitting the hyperparameter space, sampling method and termination policy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set up experiment name\n",
+    "exp_name = \"dpv2-image-classification-experiment\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "image_classification_job = automl.image_classification(\n",
+    "    compute=compute_name,\n",
+    "    # name=\"dpv2-image-classification-job-02\",\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=\"label\",\n",
+    "    primary_metric=\"accuracy\",\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    "    properties={\"enable_automode\": True},\n",
+    ")\n",
+    "\n",
+    "image_classification_job.set_limits(\n",
+    "    timeout_minutes=60, max_trials=5, max_concurrent_trials=2\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Submitting an AutoML job for Computer Vision tasks\n",
+    "Once you've configured your job, you can submit it as a job in the workspace in order to train a vision model using your training dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit the AutoML job\n",
+    "returned_job = ml_client.jobs.create_or_update(\n",
+    "    image_classification_job\n",
+    ")  # submit the job to the backend\n",
+    "\n",
+    "print(f\"Created job: {returned_job}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.2. Individual runs\n",
+    "\n",
+    "If automatic sweeps do not meet your needs, you can launch individual runs to explore a few model algorithms; we provide sensible default hyperparameters for each algorithm. You can also launch individual runs for the same model algorithm and a few hyperparameters.\n",
+    "\n",
+    "The following functions are used to configure the AutoML image job:\n",
     "### image_classification() function parameters:\n",
```

Comment:
The text about set_limits and image_classification should be moved to 4.1 section under automode. Automode also uses these apis.

Reply:
Also, need to add max_trials, max_concurrent_trials description im set_limit.

Reply:
Done.